mmv1/products/orgpolicy/api.yaml

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+--- !ruby/object:Api::Product
+name: OrgPolicy
+display_name: Organization Policy
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://orgpolicy.googleapis.com/v2/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Organization Policy API 
+    url: https://console.cloud.google.com/apis/api/orgpolicy.googleapis.com/overview
+objects:
+  - !ruby/object:Api::Resource
+    name: 'CustomConstraint'
+    create_url: '{{parent}}/customConstraints'
```

Comment:
I suspect this line is redundant now fwiw. Non-actionable, up to you to keep or remove.

We should really make the compiler complain about redundant definitions at some point 😅

Reply:
Ah good call, I updated it to remove the redundancy. Agreed that having a check for this would be awesome!